test/runtests.jl

```diff
@@ -201,10 +201,64 @@ datadir = joinpath(@__DIR__,"data")
   end
 
   @testset "save" begin
-    table = GeoTables.load(joinpath(datadir,"path.shp"))
-    GeoTables.save(joinpath(datadir,"path.geojson"), table)
-    table = GeoTables.load(joinpath(datadir,"zone.shp"))
-    GeoTables.save(joinpath(datadir,"path.geojson"), table)
+    writedir = joinpath(tempdir(), "geotables")
+    mkdir(writedir)
```

Comment:
I think we should write to the datadir that is already defined. Can we avoid creating a new directory?

Reply:
Done!